authentication-ldap/src/test/java/com/synopsys/integration/alert/authentication/ldap/LdapAuthenticationPerformerTest.java

```diff
@@ -17,53 +17,38 @@
-import org.springframework.security.ldap.userdetails.InetOrgPersonContextMapper;
 
-import com.google.gson.Gson;
 import com.synopsys.integration.alert.api.authentication.security.UserManagementAuthoritiesPopulator;
 import com.synopsys.integration.alert.api.authentication.security.event.AuthenticationEventManager;
 import com.synopsys.integration.alert.authentication.ldap.action.LdapManager;
 import com.synopsys.integration.alert.authentication.ldap.database.accessor.LDAPConfigAccessor;
-import com.synopsys.integration.alert.authentication.ldap.database.configuration.MockLDAPConfigurationRepository;
 import com.synopsys.integration.alert.authentication.ldap.model.LDAPConfigModel;
-import com.synopsys.integration.alert.common.AlertProperties;
 import com.synopsys.integration.alert.common.descriptor.accessor.RoleAccessor;
 import com.synopsys.integration.alert.common.enumeration.AuthenticationType;
-import com.synopsys.integration.alert.common.persistence.util.FilePersistenceUtil;
-import com.synopsys.integration.alert.common.security.EncryptionUtility;
-import com.synopsys.integration.alert.common.util.DateUtils;
-import com.synopsys.integration.alert.test.common.MockAlertProperties;
 
 class LdapAuthenticationPerformerTest {
-    private LDAPConfigAccessor ldapConfigAccessor;
+    private final LDAPConfigAccessor ldapConfigAccessor = LDAPTestHelper.createTestLDAPConfigAccessor();
     private LdapManager ldapManager;
     private LdapAuthenticationPerformer ldapAuthenticationPerformer;
     private Authentication inputAuthentication;
-    private static LDAPConfigModel ldapConfigModel;
+    private LDAPConfigModel validaLDAPConfigModel;
+    private LDAPConfigModel inValidLDAPConfigModel;
 
     @Mock
     UserManagementAuthoritiesPopulator MockUserManagementAuthoritiesPopulator;
     @Mock
     AuthenticationEventManager mockAuthenticationEventManager;
     @Mock
     RoleAccessor mockRoleAccessor;
-    @Mock
-    LdapAuthenticationProvider mockLdapAuthenticationProvider;
 
     @BeforeEach
     public void init() {
-        AlertProperties alertProperties = new MockAlertProperties();
-        FilePersistenceUtil filePersistenceUtil = new FilePersistenceUtil(alertProperties, new Gson());
-        EncryptionUtility encryptionUtility = new EncryptionUtility(alertProperties, filePersistenceUtil);
-        MockLDAPConfigurationRepository mockLDAPConfigurationRepository = new MockLDAPConfigurationRepository();
-        InetOrgPersonContextMapper inetOrgPersonContextMapper = new LdapConfig().ldapUserContextMapper();
-
-        ldapConfigAccessor = new LDAPConfigAccessor(encryptionUtility, mockLDAPConfigurationRepository);
-        ldapManager = new LdapManager(ldapConfigAccessor, MockUserManagementAuthoritiesPopulator, inetOrgPersonContextMapper);
+        ldapManager = new LdapManager(ldapConfigAccessor, MockUserManagementAuthoritiesPopulator, new LdapConfig().ldapUserContextMapper());
```

Comment:
The parameter 'MockUserManagementAuthoritiesPopulator' should be renamed to start with a lowercase.